### ISSUE TITLE
Error when selecting a testcase using the testcase number (CLI Argument -tc) 

### ISSUE THREAD

**Issue by maleiter** [HDLUtils/hdlregression#8]
I've a testbench with 2 test cases. 
When executing hdlregression.py (in my case hr.py) with the argument ```python hr.py -tc 2```, the output says that the index 2 is out of the range 1 to 2. 

Here is my output:

```
malei@LAPTOP-KA6BVU31 MINGW64 /c/P2L2/Open-Source/HDLRegression/Lab01/hr
$ py hr.py -ltc
Unable to read version.txt

================================================================================
  HDLRegression version 0.0.0
  See /doc/hdlregression.pdf for documentation.
================================================================================


Scanning files...
Building test suite structure...
TC:1 - tb_pwmgen_hdlregression.bhv.testcase_1_default
TC:2 - tb_pwmgen_hdlregression.bhv.testcase2_failing

malei@LAPTOP-KA6BVU31 MINGW64 /c/P2L2/Open-Source/HDLRegression/Lab01/hr
$ py hr.py -tc 2
Unable to read version.txt

================================================================================
  HDLRegression version 0.0.0
  See /doc/hdlregression.pdf for documentation.
================================================================================


Scanning files...
Building test suite structure...
Testcase index out of range (1 to 2).
```

**Comment by HDLUtils** [comment]
Hi,
Which version of HDLRegression are you running?
Are both tests run if you use the full_regression flag "py hr.py -fr"?

Br,
HU

**Comment by maleiter** [comment]
Hi, I'm using the latest main branch.
Yes, all tests run if I use ```py hr.py -fr```

I've fixed it in PR #9

**Comment by HDLUtils** [comment]
Great, thanks @maleiter :)
